code/modules/vore/eating/leave_remains_vr.dm

```diff
@@ -0,0 +1,156 @@
+	if(user.a_intent == I_HURT)
+		to_chat(user,"<span class='warning'>As you squeeze the [name], it crumbles into dust and falls apart into nothing!</span>")
+		qdel(src)
+
+/obj/item/weapon/digestion_remains/ribcage
+	name = "ribcage"
+	desc = "A bleached ribcage. It's very white and definitely has seen better times. Hard to tell what it belonged to."
+	icon_state = "ribcage"
+
+/obj/item/weapon/digestion_remains/skull
+	name = "skull"
+	desc = "A bleached skull. It looks very weakened. Seems like it belonged to a human."
+	icon_state = "skull"
+
+/obj/item/weapon/digestion_remains/skull/tajaran
+	desc = "A bleached skull. It looks very weakened. Seems like it belonged to a tajara."
+	icon_state = "skull_taj"
+
+/obj/item/weapon/digestion_remains/skull/unathi
+	desc = "A bleached skull. It looks very weakened. Seems like it belonged to an unathi."
+	icon_state = "skull_unathi"
+
+/obj/item/weapon/digestion_remains/skull/skrell
+	desc = "A bleached skull. It looks very weakened. Seems like it belonged to a skrell."
+	icon_state = "skull"
+
+/obj/item/weapon/digestion_remains/skull/vasilissan
+	desc = "A bleached skull. It looks very weakened. Seems like it belonged to a vasilissan."
+	icon_state = "skull"
+
+/obj/item/weapon/digestion_remains/skull/akula
+	desc = "A bleached skull. It looks very weakened. Seems like it belonged to an akula."
+	icon_state = "skull_unathi"
+
+/obj/item/weapon/digestion_remains/skull/rapala
+	desc = "A bleached skull. It looks very weakened. Seems like it belonged to a rapala."
+	icon_state = "skull"
+
+/obj/item/weapon/digestion_remains/skull/vulpkanin
+	desc = "A bleached skull. It looks very weakened. Seems like it belonged to a vulpkanin."
+	icon_state = "skull_taj"
+
+/obj/item/weapon/digestion_remains/skull/sergal
+	desc = "A bleached skull. It looks very weakened. Seems like it belonged to a sergal."
+	icon_state = "skull_taj"
+
+/obj/item/weapon/digestion_remains/skull/zorren
+	desc = "A bleached skull. It looks very weakened. Seems like it belonged to a zorren."
+	icon_state = "skull_taj"
+
+/obj/item/weapon/digestion_remains/skull/nevrean
+	desc = "A bleached skull. It looks very weakened. Seems like it belonged to a nevrean."
+	icon_state = "skull_taj"
+
+/obj/item/weapon/digestion_remains/skull/teshari
+	desc = "A bleached skull. It looks very weakened. Seems like it belonged to a teshari."
+	icon_state = "skull_taj"
+
+/obj/item/weapon/digestion_remains/skull/vox
+	desc = "A bleached skull. It looks very weakened. Seems like it belonged to a vox."
+	icon_state = "skull_taj"
+
+/obj/item/weapon/digestion_remains/skull/unknown
+	desc = "A bleached skull. It looks very weakened. You can't quite tell what species it belonged to."
+	icon_state = "skull"
+
+/obj/item/weapon/digestion_remains/skull/unknown/anthro
+	icon_state = "skull_taj"
+
+/obj/item/weapon/digestion_remains/skull/xenohybrid
+	desc = "A bleached skull. It looks very weakened. Seems like it belonged to something with prolonged head."
```

Comment:
"Seems like it belonged to something with an elongated head."

Reply:
Thanks. I was struggling with right word there. Changed.